com.unity.multiplayer.mlapi/Runtime/Core/NetworkedObject.cs

```diff
@@ -248,7 +248,14 @@ public void NetworkShow(ulong clientId, Stream payload = null)
         /// <param name="payload">An optional payload to send as part of the spawns</param>
         public static void NetworkShow(List<NetworkedObject> networkedObjects, ulong clientId, Stream payload = null)
         {
-            if (!NetworkingManager.Singleton.IsServer)
+            if (networkedObjects.Count == 0)
+            {
+                return;
+            }
+
+            NetworkingManager networkingManager = networkedObjects[0].NetManager;
```

Comment:
Is it guaranteed that every NetworkObject will have the same NetManager?

Reply:
@TwoTenPvP this (and the non-static NetworkShow) are not in use.  Are these worth keeping around?

If so, I think we can and should make this an instance method off of NetworkedManager.  It doesn't seem to use any static vars off of NetworkedObject, and mostly is bound to NetworkingManager anyway.

Reply:
It was very difficult for me to envision a scenario where you would hand this method a heterogenous list of NetworkObjects, belonging to multiple different NetworkManagers. It seemed like something that could be handled by specifying that as a method precondition in the method's comments, which I've added in my latest changes. 

That said, I was anticipating these methods would go away in favor of matt's new visibility work.